src/main/java/org/aion/wallet/account/AccountManager.java

```diff
@@ -53,7 +56,7 @@
 
     private final KeystoreFormat keystoreFormat = new KeystoreFormat();
 
-    private final Timer lockTimer = new Timer(true);
+    private final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
```

Comment:
Do we want to add more threads later? If not then please use `newSingleThreadScheduledExecutor()`